charts/neon-storage-controller/templates/_helpers.tpl

```diff
@@ -60,3 +60,11 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Returns service name
+This will be use only for internal purpose e.g. ingress connecting to service
+*/}}
+{{- define "neon-storage-controller.serviceName" -}}
+{{- printf "%s-svc" (include "neon-storage-controller.fullname" .) | replace "+" "_" | trunc 63 | trimSuffix "-" }}
```

Comment:
Having a service name with the `-svc` prefix would be an outlier in our service names. Maybe we should set it explicitly rather than using a helper?

Reply:
I'm defining in helper to avoid using `-svc` in multiple places. e.g. we are referring to this in ingress as well.

Reply:
(As usual I typed `prefix` when I meant `suffix` in the original comment)

Yes, but should the new service we called `neon-storage-controller-svc` for example? We usually don't use the `-svc` suffix in our service names. I realize we need a suffix so maybe it should be `-ingress` to make it explicit?

Reply:
I used suffixes as we have to avoid conflict with existing service name. I don't think it's good to add `-ingress` as well. it's a generic k8s service.

Reply:
LGTM